__tests__/app.test.js

```diff
@@ -52,12 +52,84 @@ describe.only('GET /api', () => {
+        return request(app)
+            .get(`/api/articles/3`)
+            .expect(200)
+            .then(({ body }) => {
+                expect(typeof body).toBe('object')
+                expect(body.article[0].title).toBe("Eight pug gifs that remind me of mitch")
+                expect(body.article[0].article_id).toBe(3)
+                expect(body.article[0].author).toBe("icellusedkars")
+                expect(body.article[0].body).toBe("some gifs")
+                expect(body.article[0].created_at).toBe('2020-11-03T09:12:00.000Z')
+                expect(body.article[0].votes).toBe(0)
+                expect(body.article[0].article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+            })
+           
+          })
+          test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+            return request(app)
+              .get('/api/articles/1111')
+              .expect(404)
+              .then(({body}) => {
+                expect(body.msg).toBe('article does not exist');
+              });
+          });
+          test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
+            return request(app)
+              .get('/api/articles/not-an-id')
+              .expect(400)
+              .then((response) => {
+                expect(response.body.msg).toBe('Bad request');
+              });
+    })
+})
+
+
+
+describe('GET /api/articles', () => {
+    it('returns a 200 status code and responds with an articles array of article objects', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles.length).toBe(13)
+                body.articles.forEach((article) => {
+                    expect(typeof article.author).toBe('string')
+                    expect(typeof article.title).toBe('string')
+                    expect(typeof article.article_id).toBe('number')
+                    expect(typeof article.topic).toBe('string')
+                    expect(typeof article.created_at).toBe('string')
+                    expect(typeof article.votes).toBe('number')
+                    expect(typeof article.article_img_url).toBe('string')
+                    expect(typeof article.comment_count).toBe('string')
+                }) 
+                expect(body.articles).toBeSortedBy('created_at', {
+                    descending: true,
+                    coerce: true,
+                  });
+
+            })
+    })
+    it('GET:404 sends an appropriate status and error message when given an invalid endpoint', () => {
+        return request(app)
+            .get('/api/not-a-route')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Not found');
```

Comment:
This test is doing the same thing each time you've used it.
If you seperate it into another describe block then it wouldn't need repeating each time.